test/integration/dialects/postgres/query-generator.test.js

```diff
@@ -856,6 +856,9 @@ if (dialect.match(/^postgres/)) {
           arguments: ['User', ['username', 'isAdmin'], {}, 'User'],
           expectation: 'CREATE INDEX \"user_username_is_admin\" ON \"User\" (\"username\", \"isAdmin\")'
         }, {
+          arguments: [{schema: 'mySchema', tableName: 'myTable'}, ['username', 'isAdmin'], {}, 'User'],
+          expectation: 'CREATE INDEX \"user_username_is_admin\" ON \"mySchema\".\"myTable\" (\"username\", \"isAdmin\")'
```

Comment:
https://github.com/sequelize/sequelize/blob/master/test/unit/sql/index.test.js

Could you add a test here instead? We're working on removing SQL tests from the integration test suite.